docs/multicast.md

```diff
@@ -0,0 +1,378 @@
+# Multicast
+
+## Introduction
+IP multicast enables data to be delivered to multiple IP addresses
+simultaneously.
+Multicast can distribute data one-to-many or many-to-many. For this to happen,
+the 'receivers' join a multicast group, and the sender(s) send data to it.
+In other words, multicast filtering is achieved by dynamic group control
+management.
+
+The multicast group membership is implemented with IGMP. For details, check RFCs
+[1112](https://datatracker.ietf.org/doc/html/rfc1112)
+and [2236](https://datatracker.ietf.org/doc/html/rfc2236).
+
+## Configuring multicast on the cluster
+
+### Requirements
+The multicast feature is implemented using
```

Comment:
@maiqueb I don't think we need this bit about OVN requirements. ovnkube has required PortGroup support since https://github.com/ovn-org/ovn-kubernetes/pull/1148 in March 2020.

Also, IGMP_Group has been in OVN since 20.09 and ovnkube requires at least OVN 20.12 or later.

Reply:
Fair point.

Done.